### PR TITLE
Availability Component: Get calendars using the access token

### PR DESCRIPTION
**Pull request by pooja169usp**
Get calendars using the access token. Search for is_primary: true and use the id associated with that object. Making this GET request in onMount to get calendar_id for POST /events, so there's no lag for user

# Readiness checklist

- [] Cypress tests passing?
- [] New cypress tests added?
- [] Included before/after screenshots, if the change is visual

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by wonka-lighthouse — COMMENTED** [review]
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/agenda/src/index.html |

     
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/composer/src/index.html |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: components/availability/src/Availability.svelte [hunk 1]
```diff
@@ -39,11 +44,20 @@
...
     const storeKey = JSON.stringify({ component_id: id, access_token });
     manifest = (await $ManifestStore[storeKey]) || {};
+    const calendarQuery: CalendarQuery = {
+      access_token,
+      component_id: id,
+      calendarIDs: [], // empty array will fetch all calendars
+    };
+    const calendarsList = await CalendarStore.getCalendars(calendarQuery);
+    calendarID = calendarsList.find((cal) => cal.is_primary)?.id || "";
+    console.log(calendarID);
```
Comment (ozsivanov) [inline components/availability/src/Availability.svelte:60]: ✂️

**Review by ozsivanov — APPROVED** [review]
Looks good! Please update PR title before merging.

File: components/availability/src/Availability.svelte (hunk 1 above)
Comment (pooja169usp) [reply components/availability/src/Availability.svelte:60]: This was intended. It will be removed when we add POST event for creating events

**Pull request merged**